tests/solana_tests/create_contract.rs

```diff
@@ -566,10 +566,9 @@ fn account_with_seed_bump() {
     let mut vm = build_solidity(
         r#"
         contract bar {
-
             @space(511 + 102)
             @payer(payer)
-            constructor(@seed bytes seed, @bump byte b) {}
+            constructor(@seed address seed, @seed bytes2 seed2, @bump byte b) {}
```

Comment:
When creating an account with seeds, the runtime checks if the account address can be derived with the passed seeds and bump. Our virtual machine does not do so, does it? This may mask problems in our test.

Reply:
The tests are specifically written to test that the correct PDA is created.

Reply:
From what I read in `tests/solana.rs` we do not check if the PDA being created matches the address derived with the seeds. Can you please pass in the wrong seed in this test to check if it fails?

I don't have my computer with me, and I don't want to delay this PR because of this.